docs/pages/components/pagination.md

```diff
@@ -89,11 +89,11 @@ When the first page is active, the Back arrow should be disabled.
     <a href="#" class="fd-pagination__link fd-pagination__link--previous" aria-label="Previous"
     aria-disabled="false"></a>
     <a href="#" class="fd-pagination__link">1</a>
-    <span class="fd-pagination__link fd-pagination__link--more" role="presentation"></span>
+    <span class="fd-pagination__link--more" role="presentation"></span>
     <a href="#" class="fd-pagination__link">21</a>
     <a href="#" class="fd-pagination__link" aria-selected="true">22</a>
     <a href="#" class="fd-pagination__link">23</a>
-    <span class="fd-pagination__link fd-pagination__link--more" role="presentation"></span>
+    <span class="fd-pagination__link--more" role="presentation"></span>
```

Comment:
Can we put this back to the way it was before? `"fd-pagination__link fd-pagination__link--more"` That way it follows the established patterns and `--more` would just have that extra before block